posix.mak

```diff
@@ -547,7 +547,7 @@ style: ../dscanner/dsc $(LIB)
 publictests: $(LIB)
 	# parse all public unittests from Phobos, for now some modules are excluded
 	rm -rf ./out
-	DFLAGS="$(DFLAGS) $(LIB) -defaultlib= -debuglib= $(LINKDL)" $(DUB) --compiler=$${PWD}/$(DMD) --single ../tools/phobos_tests_extractor.d -- --inputdir . --ignore "base64.d,building_blocks/free_list,building_blocks/quantizer,digest/hmac.d,file.d,index.d,math.d,ndslice/selection.d,stdio.d,traits.d,typecons.d,uuid.d" --outputdir ./out
+	DFLAGS="$(DFLAGS) $(LIB) -defaultlib= -debuglib= $(LINKDL)" $(DUB) --compiler=$${PWD}/$(DMD) --root=../tools/styles -c tests_extractor -- --inputdir . --ignore "index.d,ndslice/selection.d,traits.d,typecons.d,digest/hmac.d" --outputdir ./out
```

Comment:
I am not sure how this pseudo unittest from `std.digest.hmac` should be handled:

https://github.com/dlang/phobos/blob/master/std/digest/hmac.d#L34

(it tries to parse from stdin)